.github/workflows/build-bootc-image.yaml

```
@@ -172,10 +184,10 @@ jobs:
             docker://${OCI_IMAGE}:${OCI_IMAGE_TAG}
 
       - name: Tag manifest list with extra tags, push and sign them
-        if: ${{ inputs.tags != '' }}
+        if: ${{ steps.read-tags.outputs.all_tags != '' }}
```

actionlint (1.7.10)

property "read-tags" is not defined in object type {}

(expression)